lib/warhol/router.rb

```diff
@@ -5,19 +5,21 @@ class Router
     include ::CanCan::Ability
     include Warhol::Support::Inflector
 
-    attr_reader :user
+    attr_reader :object
```

Comment:
I think I would still leave a comment above this line that says
```
# typically a user
```
This is just to smooth the reading & orientation for some people, around the typical case.